### PR TITLE
Update manpages for 8.0

### PR DESCRIPTION
**Pull request by am11**
Used `documentation/manpages/tool/run_docker.sh` to sync manpages with latest dotnet/docs.

Targeting <del>release/8.0.2xx</del> <kbd>release/8.0.1xx</kbd> branch.

## Customer Impact

Linux packages will contain up-to-date manpages that align with the tools we actually shipped in the release.

## Risk

**Low**, only updates documentation files, and these are updated from learn.ms.com.

## Regression

No, but we may want to look into adding this to our release checklist prior to each quarterly/major release to prevent this brief lag.

### CODE REVIEW

**Comment by baronfel** [comment]
This is targeted for 8.0.2xx, but the distros don't build that version, right? So if the distros want updated man pages we'd need this to target release/8.0.1xx? It's late in the cycle but we could try to get it in for GA or the first servicing of it was important to source-build partners.

**Comment by am11** [comment]
@baronfel, I based off of latest branch of release/8 series, unaware of its purpose. :)
I've now switched to release/8.0.1xx branch.

> It's late in the cycle but we could try to get it in for GA or the first servicing of it was important to source-build partners.

Perhaps @dotnet/distro-maintainers  may know about the desired timeline? Note that changes for dotnet/docs repo pertaining to dotnet(1) and its commands are not too frequent, so it maybe alright to do it now (three weeks ahead of the release date). It would be great if this manpage update is somehow automated where we have source-build release tagging infra, so it is not forgotten. e.g. "one day before the scheduled release of next major version, dotnet bot to post this pull request in dotnet/sdk repo with changes produced by `sdk/documentation/manpages/tool/run_docker.sh` script".

**Comment by omajid** [comment]
Getting this into 8.0 would be great. I don't think it's important/urgent enough to respin the GA if that's already been finalized, though. First servicing, for example, should be just fine.

**Comment by mateusrodrigues** [comment]
+1 on @omajid's response

**Comment by baronfel** [comment]
Thanks for the clarification, folks! Let's aim for first servicing then.

File: documentation/manpages/sdk/dotnet-pack.1 [hunk 1]
```diff
@@ -164,6 +164,13 @@ Doesn\[cq]t display the startup banner or the copyright message.
 .RS 2
 .PP
 Places the built packages in the directory specified.
+.IP \[bu] 2
+\&.NET 7.0.200 SDK
+.RS 2
+.PP
+In the 7.0.200 SDK, if you specify the \f[V]--output\f[R] option when running this command on a solution, the CLI will emit an error.
```
Comment (joeloff) [inline documentation/manpages/sdk/dotnet-pack.1:171]: does this error in 7.0.100 as well? 7.0.200 shipped after .1xx

Comment (baronfel) [reply documentation/manpages/sdk/dotnet-pack.1:171]: We actually removed this warning entirely for pack in 201 - the source documentation needs updating.

Comment (baronfel) [reply documentation/manpages/sdk/dotnet-pack.1:171]: lol, ignore me - it actually says this right below.

to answer your question @joeloff it does not error in 7.0.100 - this was an inadvertent breaking change that slipped into 7.0.200 that we rightfully got lambasted for.

**Review by marcpopMSFT — APPROVED** [review]

**Pull request merged**